src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngine.java

```diff
@@ -115,17 +117,41 @@ public StateFlow build() {
             .state(CLAIM_ISSUED_PAYMENT_FAILED)
                 .transitionTo(CLAIM_ISSUED_PAYMENT_SUCCESSFUL).onlyIf(paymentSuccessful)
             .state(CLAIM_ISSUED_PAYMENT_SUCCESSFUL)
-                .transitionTo(PENDING_CLAIM_ISSUED).onlyIf(pendingClaimIssued)
-                .transitionTo(PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT).onlyIf(respondent1NotRepresented
-                                                                                       .or(respondent2NotRepresented))
-                .transitionTo(PENDING_CLAIM_ISSUED_UNREGISTERED_DEFENDANT).onlyIf(respondent1OrgNotRegistered
-                                                                                      .or(respondent2OrgNotRegistered))
+            .transitionTo(PENDING_CLAIM_ISSUED).onlyIf(pendingClaimIssued)
+//               Unrepresented
```

Comment:
I've retained the comments because i think it's useful, let me know if you think otherwise and we can discuss to remove it or not.